community/modules/scheduler/gke-cluster/variables.tf

```diff
@@ -270,3 +270,21 @@ variable "service_account" {
     error_message = "service_account is deprecated and replaced with service_account_email and scopes."
   }
 }
+variable "kubernetes_labels" {
+  description = <<-EOT
+  Kubernetes labels to be applied to each node in the node group. Key-value pairs. 
+  (The `kubernetes.io/` and `k8s.io/` prefixes are reserved by Kubernetes Core components and cannot be specified)
+  EOT
+  type        = map(string)
+  default     = null
+}
+variable "image_type" {
+  description = "The default image type used by NAP once a new node pool is being created. Use either COS_CONTAINERD or UBUNTU_CONTAINERD."
+  type        = string
+  default     = "COS_CONTAINERD"
+}
+variable "enable_secure_boot" {
```

Comment:
This module creates both the GKE cluster and the system node pool. All three of the new variables are specific to the system node pool. The other variables in this module that are specific to system node pool are prefixed with `system_node_pool_`. 

I would suggest using:
- `system_node_pool_kubernetes_labels`
- `system_node_pool_image_type`
- `system_node_pool_enable_secure_boot`

Also, move these variables to ~L173 so all the system node pool variables are listed together.